app/src/org/commcare/preferences/DeveloperPreferences.java

```diff
@@ -236,15 +227,6 @@ public void onSharedPreferenceChanged(SharedPreferences sharedPreferences, Strin
                     DevSessionRestorer.clearSession();
                 }
                 setSessionEditText();
-
-                break;
-            case LOAD_FORM_PAYLOAD_AS:
```

Comment:
Earlier we were blocking the sync on changing the load form saved status to Unsent/Incomplete by clearing out the submission url so that user doesn't end up submitting lots of form to HQ. Now since we are using this to test the form upload, we want to be able to sync forms after we load forms as unsent/incomplete.